site/content/overview/resource-validation.md

```diff
@@ -74,53 +27,44 @@ kubectl apply -f coffee-route.yaml
-> If this happens, Step 4 will reject any invalid values (from NGINX perspective).
+{{< note >}}While unlikely, bypassing this validation step is possible if the Gateway API CRDs are modified to remove the validation.If this happens, Step 4 will reject any invalid values (from NGINX perspective).{{< /note >}}
 
-### Step 2 - For Kubernetes 1.25+ - CEL Validation by Kubernetes API Server
+## Step 2 - CEL or Webhook validation by Kubernetes
 
-The Kubernetes API server validates Gateway API resources using CEL validation embedded in the Gateway API CRDs.
-It validates Gateway API resources using advanced rules unavailable in the OpenAPI schema validation.
-For example, if you create a Gateway resource with a TCP listener that configures a hostname, the CEL validation will
-reject it with the following error:
+- **Kubernetes 1.25 and later - CEL validation by Kubernetes API Server**
 
+   The Kubernetes API server validates Gateway API resources using CEL validation embedded in the Gateway API CRDs. It validates Gateway API resources using advanced rules unavailable in the OpenAPI schema validation. For example, if you create a Gateway resource with a TCP listener that configures a hostname, the CEL validation will reject it with the following error:
 
-```shell
-kubectl apply -f some-gateway.yaml
-```
+   ```shell
+   kubectl apply -f some-gateway.yaml
+   ```
 
-```text
-The Gateway "some-gateway" is invalid: spec.listeners: Invalid value: "array": hostname must not be specified for protocols ['TCP', 'UDP']
-```
+   ```text
+   The Gateway "some-gateway" is invalid: spec.listeners: Invalid value: "array": hostname must not be specified for protocols ['TCP', 'UDP']
+   ```
 
-More information on CEL in Kubernetes can be found [here](https://kubernetes.io/docs/reference/using-api/cel/).
+   More information on CEL in Kubernetes can be found [here](https://kubernetes.io/docs/reference/using-api/cel/).
 
-### Step 2 - For Kubernetes 1.23 and 1.24 - Webhook Validation by Gateway API Webhook
 
-The Gateway API comes with a validating webhook which is enabled by default in the Gateway API installation manifests.
-It validates Gateway API resources using advanced rules unavailable in the OpenAPI schema validation. For example, if
-you create a Gateway resource with a TCP listener that configures a hostname, the webhook will reject it with the
-following error:
+- **Kubernetes 1.23 and 1.24 - Webhook validation by Gateway API Webhook**
 
-```shell
-kubectl apply -f some-gateway.yaml
-```
+   The Gateway API comes with a validating webhook which is enabled by default in the Gateway API installation manifests. It validates Gateway API resources using advanced rules unavailable in the OpenAPI schema validation. For example, if you create a Gateway resource with a TCP listener that configures a hostname, the webhook will reject it with the following error:
```

Comment:
This is no longer true. The user has to install the webhook separately now if they are running on k8s 1.23 or 1.24.

Reply:
@kate-osborn Does installing the webhook separately already have instructions elsewhere that can be linked to?

It sounds like this could probably be broken into two sections based on the version they're running: it's dependent on if the expectation is that the reader is always running the latest available versions.

Reply:
@Jcahilltorre my suggestion did not include the actual link to the installation doc. It was a placeholder since I wasn't sure how to format the link. The instructions are here: https://github.com/nginxinc/nginx-gateway-fabric/blob/main/site/content/includes/installation/install-gateway-api-resources.md

Reply:
My mistake on accepting the suggestion without thorough review. Please, going forward, only use the suggestion option for ready-to-commit suggestions. If changes are required use a regular comment and call out the bits that need to be completed.